app/javascript/components/pages/SevenWondersSlides.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import {
+  Carousel,
+  CarouselItem,
+  CarouselControl,
+  CarouselIndicators,
+  CarouselCaption
+} from 'reactstrap';
+import wonders from './SevenWondersInfo';
+
+class SevenWondersSlides extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { activeIndex: 0 }
+    this.next = this.next.bind(this)
+    this.previous = this.previous.bind(this)
+    this.goToIndex = this.goToIndex.bind(this)
+    this.onExiting = this.onExiting.bind(this)
+    this.onExited = this.onExited.bind(this)
+  }
+
+  onExiting() {
+    this.animating = true
+  }
+
+  onExited() {
+    this.animating = false
+  }
+
+  next() {
+    if (this.animating) return;
+    const nextIndex = this.state.activeIndex === wonders.length - 1 ? 0 : this.state.activeIndex + 1;
+    this.setState({ activeIndex: nextIndex })
+  }
+
+  previous() {
+    if (this.animating) return;
+    const nextIndex = this.state.activeIndex === 0 ? wonders.length - 1 : this.state.activeIndex - 1;
+    this.setState({ activeIndex: nextIndex })
+  }
+
+  goToIndex(newIndex) {
+    if (this.animating) return;
+    this.setState({ activeIndex: newIndex })
+  }
+
+  render() {
+    const { activeIndex } = this.state
+
+    const slides = wonders.map((item) => {
+      return (
+        <CarouselItem
+          onExiting={this.onExiting}
+          onExited={this.onExited}
+          key={item.id}
+        >
+          <img 
+            src={item.image} 
+            alt={item.name} 
+            style={{height:"78vh",width:"100%",opacity:"90%",}}
+            />
+          <CarouselCaption 
+          className='carouselsub'
+          captionText={item.location} 
+          />
+          <CarouselCaption 
+          className='carouselheader'
+          captionHeader={item.name} 
+          />
+        </CarouselItem>
+      )
+    })
+
+    return (
+      <Carousel
+        activeIndex={activeIndex}
+        next={this.next}
+        previous={this.previous}
+      >
+        <CarouselIndicators items={wonders} activeIndex={activeIndex} onClickHandler={this.goToIndex} />
+        {slides}
+        <CarouselControl items={wonders} direction="prev" directionText="Previous" onClickHandler={this.previous} />
+        <CarouselControl items={wonders} direction="next" directionText="Next" onClickHandler={this.next} />
```

Comment:
I would recommend keeping your formatting consistent.  The attributes are dropped to a new line on most of your other tags, so I would recommend doing the same here.